src/styles/scss/card.scss

```
@@ -4,24 +4,38 @@
   font-weight: 400;
 }
 
+$border-color: #f3f3f3;
+
 .card {
   width: 200px;
-  padding: 30px 14px 14px;
+  padding: 0 14px 14px;
   display: flex;
   flex-direction: column;
   background: #fff;
-  border: 1px solid #f3f3f3;
+  border: 1px solid $border-color;
   border-radius: 5px;
   margin-bottom: 51px;
 
+  &__box-image {
+    display: flex;
+    justify-content: center;
+    padding-top: 14px;
+    margin: 13px 0 42px;
+    border: 3px solid transparent;
+    transition: transform 1s linear;
+
+    &:hover {
+      transform: scale(1.2);
+      border: 3px $border-color solid;
```

```suggestion
      border: 3px solid $border-color;
```
use same order as on the 24-th line